src/js/components/CommitId.jsx

```diff
@@ -18,10 +18,18 @@ export class CommitId extends Component {
         } else {
             displayValue = '–';
         }
+
+        if (url) {
+            return (<a href={url} target="_blank" title="Opens commit in a new window">
```

Comment:
I would suggest adding an optional "title" prop that defaults to this string value. We want to make sure we can apply i18n here.